packages/app/src/subpages/notebookSelector/components/IconButton.tsx

```
@@ -0,0 +1,23 @@
+  tooltip,
+}: {
+  icon: React.ElementType;
+  onClick: () => void;
+  tooltip: string;
+}) {
+  return (
+    <Tooltip content={tooltip}>
+      <button
+        className="opacity-35 hover:opacity-60 transition-opacity duration-200 cursor-pointer"
+        onClick={onClick}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)